src/Goerli-DssSpell.t.sol

```diff
@@ -496,7 +497,7 @@ contract DssSpellTest is GoerliDssSpellTestBase {
         assertEq(vest.rxd(1), WAD);
     }
 
-    function testVestDAIFails() public {
+    function testVestDAIFails() private { // make public to use
```

Comment:
This test can actually be removed as was very specific of last spell.

Reply:
Done.